README.md

```diff
@@ -155,29 +155,29 @@ Outer product a.k.a. tensoring expands all possible products in two arrays into
-| Division               | `6 % 2`      | `3`      | `P19 - P8`       | `P12`   |
-| Division               | `220 hz ÷ 2` | `110 Hz` | `A=3 - P8`       | `A=2`   |
-| Reduction              | `5 rd 2`     | `5/4`    | `M17^5 mod P8`   | `M3^5`  |
-| Ceiling reduction      | `2 rdc 2`    | `2`      | `P8 modc P8`     | `P8`    |
-| Exponentiation         | `3 ^ 2`      | `9`      | `P12 * 2`        | `M23`   |
-| Recipropower           | `9 /^ 2`     | `3`      | `M23 % 2`        | `P12`   |
-| Root taking            | `9 ^/ 2`     | `3`      | `M23 % 2`        | `P12`   |
-| Logarithm (in base of) | `9 /_ 3`     | `2`      | `M23 % P12`      | `2`     |
-| Round (to power of)    | `5 by 2`     | `4`      | `M17^5 to P8`    | `P15`   |
-| N of EDO               | `(5+2)\12`   | `7\12`   | _N/A_            |         |
-| NEDJI Projection       | `sqrt(2)<3>` | `1\2<3>` | _N/A_            |         |
-| Val product            | `12@ · 3/2`  | `7`      | `<12 19] dot P5` | `7`     |
+| Name                   | Linear         | Result   | Logarithmic      | Result     |
+| ---------------------- | -------------- | -------- | ---------------- | ---------- |
+| Addition               | `3 + 5`        | `8`      | _N/A_            |            |
+| Subtraction            | `5 - 3`        | `2`      | _N/A_            |            |
+| Modulo                 | `5 mod 3`      | `2`      | _N/A_            |            |
+| Ceiling modulo         | `0 modc 3`     | `3`      | _N/A_            |            |
+| Round (to multiple of) | `5 to 3`       | `6`      | _N/A_            |            |
+| Minimum                | `2 min 1`      | `1`      | `P8 min P1`      | `P1`       |
+| Maximum                | `2 max 1`      | `2`      | `P8 max P1`      | `P8`       |
+| Multiplication         | `2 * 3`        | `6`      | `P8 + P12`       | `P19`      |
+| Multiplication         | `110 Hz × 5`   | `550 Hz` | `A♮2 + M17^5`    | `C♯5^5`    |
+| Division               | `6 % 2`        | `3`      | `P19 - P8`       | `P12`      |
+| Division               | `220 hz ÷ 2`   | `110 Hz` | `A=3 - P8`       | `A=2`      |
+| Reduction              | `5 rd 2`       | `5/4`    | `M17^5 mod P8`   | `M3^5`     |
+| Ceiling reduction      | `2 rdc 2`      | `2`      | `P8 modc P8`     | `P8`       |
+| Exponentiation         | `3 ^ 2`        | `9`      | `P12 * 2`        | `M23`      |
+| Recipropower           | `9 /^ 2`       | `3`      | `M23 % 2`        | `P12`      |
+| Root taking            | `9 ^/ 2`       | `3`      | `M23 % 2`        | `P12`      |
+| Logarithm (in base of) | `9 /_ 3`       | `2`      | `M23 % P12`      | `2`        |
+| Round (to power of)    | `5 by 2`       | `4`      | `M17^5 to P8`    | `P15`      |
+| N of EDO               | `(5+2)\12`     | `7\12`   | _N/A_            |            |
+| NEDJI Projection       | `sqrt(2) ed 3` | `1\2<3>` | `2\3 ed S3`      | `2\3<9/8>` |
```

Comment:
Is it that obvious though what `sqrt(2) ed 3` does? (I know it’s not new, I just rethought it.) If `n\m` gives a monzo, then `ed` is really interesting (if not, then it’s clearer one way and less clear when one expects a monzo to be made right then).

Otherwise it seems okay!

Reply:
It's not a very obvious operation. I should add a few more words about it. (But as you said, maybe not in the scope of this PR.)